crates/client/src/client/schema/tx/transparent_receipt.rs

```diff
@@ -72,11 +72,14 @@ impl TryFrom<Receipt> for fuel_tx::Receipt {
     fn try_from(schema: Receipt) -> Result<Self, Self::Error> {
         Ok(match schema.receipt_type {
             ReceiptType::Call => fuel_tx::Receipt::Call {
-                id: schema.id.map(|id| id.into()).unwrap_or_default(),
+                id: schema
+                    .id
+                    .map(Into::into)
+                    .unwrap_or(fuel_tx::ContractId::zeroed()),
```

Comment:
Why do we no longer use `unwrap_or_default()`?

Reply:
I prefer an explicit way if it is related to business logic. In the case if default implementation is changed. On `fuel-vm` side we use `ContractId::zeroed()`.